RInAppMessaging/Classes/Models/UserInfoProvider.swift

```diff
@@ -0,0 +1,75 @@
+/**
+ * Interface which client app should implement in order for InAppMessaging SDK to get information
+ * when needed.
+ */
+@objc public protocol UserInfoProvider {
+
+    /**
+     * Only return ID token if user is logged in. Else return null.
+     *
+     * @return String of ID token.
+     */
+    func getIDToken() -> String?
```

Comment:
Why is it specifically ID token?
Should it be a generic `getAuthToken()` or just `getToken()`?